src/localization/index.js

```diff
@@ -20,11 +20,16 @@
 
 import en from './en';
 import fi from './fi';
+import pt from './pt';
 import {flattenMessages} from 'utils/locale-utils';
+import {LOCALE_CODES} from './locales';
 
 const en_flat = flattenMessages(en);
 
 export const messages = {
   en: en_flat,
-  fi: {...en_flat, ...flattenMessages(fi)}
+  fi: {...en_flat, ...flattenMessages(fi)},
```

Comment:
this is fine now, I would change it to arrar.reduce so no future addition will be needed

Reply:
But how would we be able to get the import variable value in this reduce function?

Reply:
We would need to pass imported variables **fi, pt etc...** to _flattenMessages_ function

Reply:
you can use `[key]: {...en_flat, ...flattenMessages(require(`./{key}`))},`

Reply:
cool :D thanks